app/src/pages/admin/serverSettingsPage/serverSettingsTabs/linksAndBrandingTab/addLinkForm/addLinkForm.jsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { defineMessages, useIntl } from 'react-intl';
+import { reduxForm } from 'redux-form';
+import { FieldProvider } from 'components/fields';
+import { FieldErrorHint } from 'components/fields/fieldErrorHint';
+import classNames from 'classnames/bind';
+import { FieldText } from 'componentLibrary/fieldText';
+import React from 'react';
+import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import { BigButton } from 'components/buttons/bigButton';
+import { commonValidators } from 'common/utils/validation';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { updateServerFooterLinksAction } from 'controllers/appInfo';
+import { email } from 'common/utils/validation/validate';
+import { NOTIFICATION_TYPES, showNotification } from 'controllers/notification';
+import styles from './addLinkForm.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  linkName: {
+    id: 'AddLinkForm.linkName',
+    defaultMessage: 'Link name',
+  },
+  urlOrEmail: {
+    id: 'AddLinkForm.urlOrEmail',
+    defaultMessage: 'URL or Email',
+  },
+  addedFooterLinkSuccess: {
+    id: 'LinksAndBrandingTab.addedFooterLinkSuccess',
+    defaultMessage: 'The link has been added successfully',
+  },
+});
+
+const AddLink = ({ onClose, handleSubmit, customLinks }) => {
+  const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
+  const handleAddLink = (values) => {
+    customLinks.push({ ...values, url: email(values.url) ? `mailto:${values.url}` : values.url });
+
+    try {
+      dispatch(updateServerFooterLinksAction(customLinks));
+      dispatch(
+        showNotification({
+          message: formatMessage(messages.addedFooterLinkSuccess),
+          type: NOTIFICATION_TYPES.SUCCESS,
+        }),
+      );
+    } catch (error) {
+      dispatch(
+        showNotification({
+          message: formatMessage(COMMON_LOCALE_KEYS.SOMETHING_WENT_WRONG),
+          type: NOTIFICATION_TYPES.ERROR,
+        }),
+      );
+    }
+
+    onClose();
```

Comment:
Looks like the list with links should be updated and form unmounted once the request successfully processed by backend side.
So until that the form should be present.